digitalocean/Firewall.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+from .baseapi import BaseAPI, POST, DELETE, PUT
+
+
+class Firewall(BaseAPI):
+    def __init__(self, *args, **kwargs):
+        self.id = None
+        self.status = None
+        self.created_at = None
+        self.pending_changes = []
+        self.name = None
+        self.inbound_rules = None
```

Comment:
I wonder if we shouldn't use a new class for the inbound and outbound rules similar to what was done in the load balancer: https://github.com/koalalorenzo/python-digitalocean/blob/master/digitalocean/LoadBalancer.py#L26

The json that's required to pass in is a bit complex. Eg:

```
    {
      "protocol": "tcp",
      "ports": "80",
      "sources": {
        "load_balancer_uids": [
          "4de7ac8b-495b-4884-9a69-1050c6793cd6"
        ]
      }
    },
    {
      "protocol": "tcp",
      "ports": 22,
      "sources": {
        "tags": [
          "gateway"
        ],
        "addresses": [
          "18.0.0.0/8"
        ]
      }
    }
```

https://developers.digitalocean.com/documentation/v2/#firewalls

Reply:
Should I revert? :S